doc/api/unit/unit-formatter.md

```diff
@@ -0,0 +1,56 @@
+## .unitFormatter( unit, options ) ➜ function( value )
+
+Returns a function that formats a unit according to the given unit, options, and the
+default/instance locale.
+
+The returned function is invoked with one argument: the number `value` to
+be formatted.
+
+### Parameters
+
+**unit**
+
+String value indicating the unit to be formatted. eg. "day", "week", "month", etc.
+Could also be a compound unit, eg. "mile-per-hour" or "mile/hour"
+
+**options**
+
+- form: [String] eg. "long", "short" or "narrow".
```

Comment:
Shoot. Good point.

I would add a `numberFormatter` option that takes a formatter as option.